### PR TITLE
US 1889327: Suggestion/warning fixes.

### PR DESCRIPTION
**Pull request by poojapoojari**
## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: dotnet-desktop-guide/framework/wpf/advanced/how-to-alter-the-typography-of-text.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 ---
-title: How to Alter the Typography of Text
+title: "How to Alter the Typography of Text"
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/advanced/how-to-alter-the-typography-of-text.md:2]: I just noticed that the titles aren't supposed to be title case anymore. So unless it's a proper noun or code type, it should just simply be
```suggestion
title: "How to alter the typography of text"
```

Can you alter the other files you've changed?

**Review by adegeo — CHANGES_REQUESTED** [review]

File: dotnet-desktop-guide/framework/wpf/advanced/how-to-alter-the-typography-of-text.md (hunk 1 above)
Comment (poojapoojari) [reply dotnet-desktop-guide/framework/wpf/advanced/how-to-alter-the-typography-of-text.md]: @adegeo Thank you for the suggestion! I have updated the content for the same. Please do have a look. :)

**Comment by poojapoojari** [comment]
@adegeo As per the process, we've been updating "ms.date" tag whenever an article is updated/edited. However, as per your request, I have retained the previous date for all the files. Please do have a look.
Also moving forward, do we plan to not update the "ms.date" tag for all the other files? Please confirm. :)

**Comment by adegeo** [comment]
@poojapoojari Correct. I wouldn't update any date of the article just for metadata updates.

**Review by adegeo — APPROVED** [review]

**Pull request merged**